### ISSUE TITLE
Remove years under license header files

### ISSUE THREAD

**Issue by michael-valdron** [devfile/api#1240]
/kind epic

## Which area/kind this issue is related to?

/area releng

## Issue Description

~The tasks performed for #1004 should be automated process, this process would run annually at the start of the new year to remove the need to do this manually on a yearly basis.~

The copyright years are not required under the license headers: https://github.com/devfile/api/issues/1240#issuecomment-1717228954. Removing the copyright year will make the tasks performed for #1004 no longer needed on an annual basis.

## Acceptance Criteria
- [x] #1241
- [x] #1242
- [x] #1243
- [x] #1244
- [x] #1250
- [x] ~#1252~ 
- [x] #1104

**Comment by michael-valdron** [comment]
Should update acceptance criteria to include Devfile API on completion of #1104.

**Comment by thepetk** [comment]
Out of curiosity, is this an issue related to alizer too?

**Comment by michael-valdron** [comment]
> Out of curiosity, is this an issue related to alizer too?

@thepetk Yeah this would include alizer too, good catch!

**Comment by rm3l** [comment]
I just found an article titled `Copyright Notices for Open Source Projects` on The Source. It mentions:

"**
[...]
Method One: State “Copyright the XYZ Authors” (or similar formulation — see below) where XYZ is the project name. Under this approach, the project does not make use of traditional individual and company-specific copyright notices.

Method Two: State “Copyright Red Hat”.

For either of these two methods, don’t bother including years, copyright symbols (i.e., © or approximations like (C)), or (for the second method) the “Inc.” in the Red Hat corporate name.**"

So I am wondering if this needs to be automated if there is no need to bother including the year.. Doing it only once should be fine then, I guess. WDYT?

**Comment by kim-tsao** [comment]
Also to note, if a year is included, it's only the modified file that gets updated with the year it was changed.  Untouched files do not need to get updated

**Comment by thepetk** [comment]
> So I am wondering if this needs to be automated if there is no need to bother including the year.. Doing it only once should be fine then, I guess. WDYT?

+1 to use the `Copyright Red Hat` way as it is also the preferred one inside the source guidelines where it is clearly stated that "inc" and year are not necessary fields for open source copyrights.

I've opened 4 PRs for:
- [Devfile Web](https://github.com/devfile/api/issues/1241): https://github.com/devfile/devfile-web/pull/95
- [Devfile Library](https://github.com/devfile/api/issues/1244): https://github.com/devfile/library/pull/182
- [Registry Support](https://github.com/devfile/api/issues/1242): https://github.com/devfile/registry-support/pull/185
- [Alizer](https://github.com/devfile/api/issues/1250): https://github.com/devfile/alizer/pull/26

**Comment by michael-valdron** [comment]
> > So I am wondering if this needs to be automated if there is no need to bother including the year.. Doing it only once should be fine then, I guess. WDYT?
> 
> +1 to use the `Copyright Red Hat` way as it is also the preferred one inside the source guidelines where it is clearly stated that "inc" and year are not necessary fields for open source copyrights.
> 
> I've opened 4 PRs for:
> 
>     * [Devfile Web](https://github.com/devfile/api/issues/1241): [[Devtools week] Update license headers devfile-web#95](https://github.com/devfile/devfile-web/pull/95)
> 
>     * [Devfile Library](https://github.com/devfile/api/issues/1244): [[Devtools week] Update license headers library#182](https://github.com/devfile/library/pull/182)
> 
>     * [Registry Support](https://github.com/devfile/api/issues/1242): [[Devtools week] Update license headers registry-support#185](https://github.com/devfile/registry-support/pull/185)
> 
>     * [Alizer](https://github.com/devfile/api/issues/1250): [[Devtools week] Create license header doc alizer#26](https://github.com/devfile/alizer/pull/26)

We should update the issue titles and descriptions to reflect these changes.

**Comment by michael-valdron** [comment]
> > > So I am wondering if this needs to be automated if there is no need to bother including the year.. Doing it only once should be fine then, I guess. WDYT?
> > 
> > 
> > +1 to use the `Copyright Red Hat` way as it is also the preferred one inside the source guidelines where it is clearly stated that "inc" and year are not necessary fields for open source copyrights.
> > I've opened 4 PRs for:
> > ```
> > * [Devfile Web](https://github.com/devfile/api/issues/1241): [[Devtools week] Update license headers devfile-web#95](https://github.com/devfile/devfile-web/pull/95)
> > 
> > * [Devfile Library](https://github.com/devfile/api/issues/1244): [[Devtools week] Update license headers library#182](https://github.com/devfile/library/pull/182)
> > 
> > * [Registry Support](https://github.com/devfile/api/issues/1242): [[Devtools week] Update license headers registry-support#185](https://github.com/devfile/registry-support/pull/185)
> > 
> > * [Alizer](https://github.com/devfile/api/issues/1250): [[Devtools week] Create license header doc alizer#26](https://github.com/devfile/alizer/pull/26)
> > ```
> 
> We should update the issue titles and descriptions to reflect these changes.

Done.

**Comment by michael-valdron** [comment]
All child items completed, closing this epic now.
